docs/bibliography.bib

```diff
@@ -397,7 +397,7 @@ @article{Koop2002
 
 @article{Lehtinen2007,
   title = {Estimating nucleation rates from apparent particle formation rates and vice versa: Revised formulation of the Kerminen–Kulmala equation},
-  author = {Kari E.J. Lehtinen and Miikka {Dal Maso} and Markku Kulmala and Veli-Matti Kerminen},
+  author = {Kari E.J. Lehtinen and Miikka Dal Maso and Markku Kulmala and Veli-Matti Kerminen},
```

Comment:
This is not correct: it changes the last name "Dal Maso" to "Maso" with "Dal" as a middle name. The correct way to handle this is to use the "Last, First" format.

```
@article{Lehtinen2007,
  title = {Estimating nucleation rates from apparent particle formation rates and vice versa: Revised formulation of the Kerminen–Kulmala equation},
  author = {Lehtinen, Kari E.J. and Dal Maso, Miikka and Kulmala, Markku and Kerminen, Veli-Matti},
  journal = {Journal of Aerosol Science},
  volume = {38},
  number = {9},
  pages = {988-994},
  year = {2007},
  doi = {10.1016/j.jaerosci.2007.06.009}
}
```